tests/Support/SupportCollectionTest.php

```diff
@@ -4963,6 +4963,42 @@ public function testCollect($collection)
         ], $data->all());
     }
 
+    /**
+     * @dataProvider collectionClassProvider
+     */
+    public function testDot($collection)
+    {
+        $data = $collection::make([
+            'name' => 'Taylor',
+            'meta' => [
+                'foo' => 'bar',
```

Comment:
Missing test coverage when the values themselves are also collections.

Reply:
d46028748